src/sandstorm/fuse.c++

```diff
@@ -511,49 +492,51 @@ private:
             }
 
             reply->body.generation = 0;
-            reply->newObject = CapToInsert {
-                IdType::NODE, reply->body.nodeid, lookupResult.getNode() };
+            reply->newObject = ObjToInsert(reply->body.nodeid, kj::mv(lookupResult->node));
 
             translateAttrs(attributes, &reply->body.attr);
             if (options.cacheForever) {
               reply->body.entry_valid = 365 * kj::DAYS / kj::SECONDS;
               reply->body.attr_valid = 365 * kj::DAYS / kj::SECONDS;
             } else {
-              splitTime(lookupResult.getTtl(),
+              splitTime(lookupResult->ttl,
                   &reply->body.entry_valid, &reply->body.entry_valid_nsec);
-              splitTime(attrResult.getTtl(),
+              splitTime(result.ttl,
                   &reply->body.attr_valid, &reply->body.attr_valid_nsec);
             }
-
             return kj::mv(reply);
-          });
-        }));
-
+          } else {
+            auto reply = kj::heap<ResponseBase>();
+            reply->header.error = -ENOENT;  // Has to be negative. Just because.
+            return kj::mv(reply);
+          }
+        });
         break;
       }
 
-      case FUSE_GETATTR:
-        addReplyTask(header.unique, EIO, node.getAttributesRequest(capnp::MessageSize {4, 0}).send()
-            .then([this](auto&& response) -> kj::Own<ResponseBase> {
+      case FUSE_GETATTR: {
```

Comment:
Not a showstopper by any means, but just because I'm curious and want to learn more C++: What's the meaning of this `{` here? Does it add something to this `case`?